internal/chain/transaction.go

```diff
@@ -3,16 +3,24 @@ package chain
 import (
 	"context"
 	"crypto/ed25519"
+	"crypto/sha256"
 	"encoding/binary"
 	"math/big"
 
+	primproto "buf.build/gen/go/astria/astria/protocolbuffers/go/astria/primitive/v1"
+	sqproto "buf.build/gen/go/astria/astria/protocolbuffers/go/astria/sequencer/v1alpha1"
 	client "github.com/astriaorg/go-sequencer-client/client"
-	sqproto "github.com/astriaorg/go-sequencer-client/proto"
 	"github.com/cometbft/cometbft/libs/bytes"
 
 	"github.com/ethereum/go-ethereum/common"
 )
 
+const DEFAULT_ASTRIA_ASSET = "nria"
```

Comment:
you can use the const/var in go-sequencer-client/client directly, no need to redefine